src/test/resources/org/broadinstitute/hellbender/tools/BQSR/expected.MultiSite.reads.pipeline.vcf

```diff
@@ -28,15 +28,12 @@
-20	1000386	.	T	C	908.77	.	AC=2;AF=1.00;AN=2;DP=23;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=25.36;RAW_MQ=74573.00;SOR=0.776	GT:AD:DP:GQ:PL	1/1:0,23:23:72:937,72,0
-20	1000420	.	T	C	947.77	.	AC=2;AF=1.00;AN=2;DP=21;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=28.73;RAW_MQ=54570.00;SOR=0.784	GT:AD:DP:GQ:PL	1/1:0,21:21:66:976,66,0
-20	1000442	.	TTTCA	T	727.73	.	AC=2;AF=1.00;AN=2;DP=16;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=30.97;RAW_MQ=33811.00;SOR=0.693	GT:AD:DP:GQ:PL	1/1:0,16:16:51:765,51,0
-20	1000838	.	G	C	1764.77	.	AC=2;AF=1.00;AN=2;DP=60;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=29.91;RAW_MQ=197215.00;SOR=1.217	GT:AD:DP:GQ:PL	1/1:0,59:59:99:1793,177,0
-20	1001760	.	T	C	2381.77	.	AC=2;AF=1.00;AN=2;DP=79;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=30.15;RAW_MQ=281641.00;SOR=0.718	GT:AD:DP:GQ:PL	1/1:0,79:79:99:2410,237,0
-20	1001805	.	C	T	1870.77	.	AC=2;AF=1.00;AN=2;DP=61;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=30.67;RAW_MQ=215138.00;SOR=0.793	GT:AD:DP:GQ:PL	1/1:0,61:61:99:1899,183,0
-20	1001987	.	C	G	2144.77	.	AC=2;AF=1.00;AN=2;DP=71;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=30.21;RAW_MQ=250610.00;SOR=2.082	GT:AD:DP:GQ:PL	1/1:0,71:71:99:2173,213,0
-20	1002042	.	C	A	1161.77	.	AC=2;AF=1.00;AN=2;DP=39;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;QD=29.79;RAW_MQ=137641.00;SOR=2.362	GT:AD:DP:GQ:PL	1/1:0,39:39:99:1190,117,0
+20	1000072	.	A	G	2092.77	.	AC=2;AF=1.00;AN=2;DP=69;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=53.65;QD=30.33;SOR=0.980	GT:AD:DP:GQ:PL	1/1:0,69:69:99:2121,207,0
+20	1000152	.	C	T	1310.77	.	AC=2;AF=1.00;AN=2;DP=44;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=53.11;QD=29.79;SOR=0.784	GT:AD:DP:GQ:PL	1/1:0,44:44:99:1339,132,0
+20	1000179	.	A	C	1387.77	.	AC=2;AF=1.00;AN=2;DP=47;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=54.23;QD=29.53;SOR=1.251	GT:AD:DP:GQ:PL	1/1:0,47:47:99:1416,141,0
+20	1000303	.	T	C	1261.77	.	AC=2;AF=1.00;AN=2;DP=42;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.45;QD=30.04;SOR=3.365	GT:AD:DP:GQ:PL	1/1:0,42:42:99:1290,126,0
+20	1000838	.	G	C	1764.77	.	AC=2;AF=1.00;AN=2;DP=60;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=57.33;QD=29.91;SOR=1.217	GT:AD:DP:GQ:PL	1/1:0,59:59:99:1793,177,0
+20	1001760	.	T	C	2381.77	.	AC=2;AF=1.00;AN=2;DP=79;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.71;QD=30.15;SOR=0.718	GT:AD:DP:GQ:PL	1/1:0,79:79:99:2410,237,0
+20	1001805	.	C	T	1870.77	.	AC=2;AF=1.00;AN=2;DP=61;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.39;QD=30.67;SOR=0.793	GT:AD:DP:GQ:PL	1/1:0,61:61:99:1899,183,0
+20	1001987	.	C	G	2144.77	.	AC=2;AF=1.00;AN=2;DP=71;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.41;QD=30.21;SOR=2.082	GT:AD:DP:GQ:PL	1/1:0,71:71:99:2173,213,0
+20	1002042	.	C	A	1161.77	.	AC=2;AF=1.00;AN=2;DP=39;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.41;QD=29.79;SOR=2.362	GT:AD:DP:GQ:PL	1/1:0,39:39:99:1190,117,0
```

Comment:
The fact that no `HaplotypeCallerSpark` integration tests needed to change as a result of this PR indicates that our tests for the tool are way too permissive -- can you file a ticket to implement better/more sensitive tests that would, eg., fail if someone reverted this PR?

Reply:
#4302